docs/advanced/stack-config.md

```diff
@@ -268,6 +269,206 @@ volumes:
+
+### MYSQL_USER
+
+`Default:  user`
+
+This is used to create a user, and the user is granted superuser permissions for the database specified by the `MYSQL_DATABASE` variable. Both `MYSQL_USER` and `MYSQL_PASSWORD` are required for a user to be created; if any of the two variables is not set, the other is ignored.
+
+### MYSQL_PASSWORD
+
+`Default:  user`
+
+This is used to set the newly created user's password.
+
+### MYSQL_PORT_MAPPING
+
+`Default:  3306`
+
+The port mapping to use for MySQL container. E.g. `33061:3306` will expose `3306` port as `33061` on the host. 
+
+### POSTGRES_DB
+
+`Default:  user`
+
+This variable allows you to specify the name of a database to be created on image startup
+
+### POSTGRES_USER
+
+`Default:  user`
+
+This is used to create a user, and the user is granted superuser permissions for the database specified by the `POSTGRES_DB` variable.
+
+### POSTGRES_PASSWORD
+
+`Default:  user`
+
+This is used to set the newly created user's password.
+
+### PGSQL_PORT_MAPPING
+
+`Default:  5432`
+
+The port to use when setting up PostgreSQL.
+
+### APACHE_BASIC_AUTH_USER
+
+Username to use for basic authentication.
+
+### APACHE_BASIC_AUTH_PASS
+
+Password to use for basic authentication.
+
+### HOST_UID
+
+User ID for the Container User. On MacOS & Linux defaults to current user account `id -u`.
+
+**WARNING: do not override this variable unless you know what you are doing.**
+
+### HOST_GID
+
+Group ID for the Container User. On MacOS & Linux defaults to current group account `id -g`.
+
+**WARNING: do not override this variable unless you know what you are doing.**
+
+### XDEBUG_ENABLED
+
+`Default:  0`
+
+Enables PHP XDebug Service for debugging. See [XDebug](../tools/xdebug.md)
+
+### SECRET_SSH_PRIVATE_KEY
+
+Use to pass an additional private SSH key. The key is stored in `/home/docker/.ssh/id_rsa` inside `cli` and will be considered by the SSH client in addition to the keys loaded in `docksal-ssh-agent` when establishing a SSH connection from within `cli`.
+
+### SECRET_ACAPI_EMAIL
+
+Acquia Cloud API Email Address. See [Acquia Drush Commands](../tools/acquia-drush.md)
+
+### SECRET_ACAPI_KEY
+
+Acquia Cloud API Key. See [Acquia Drush Commands](../tools/acquia-drush.md)
+
+### SECRET_TERMINUS_TOKEN
+
+Token used for logging in to Pantheon's CLI Tool [Terminus](../tools/terminus.md).
+
+## CI Variables
+
+The following variables should only be used within a CI system. They are primarily used for setting up the ability for Docksal to turn off and conserve resources.
+
+### PROJECT_INACTIVITY_TIMEOUT
+
+Defines the timeout of inactivity after which the project stack will be stopped (e.g. 0.5h)
+
+### PROJECT_DANGLING_TIMEOUT
+
+Defines the timeout of inactivity after which the project stack and code base will be entirely wiped out from the host (e.g. 168h). This requires PROJECTS_ROOT to be set.
+
+**WARNING: use at your own risk!**
+
+### PROJECTS_ROOT
+
+Used to specify the project directory root.
```

Comment:
Let's change it to `Contains path to the project root directory.`, otherwise everything looks fine to me.